commands/import.go

```diff
@@ -59,7 +59,7 @@ func doImport(c *cli.Context) error {
 
 	fmt.Printf("Importing %s\n", res.System.BroConfig.LogPath)
 	importer := parser.NewFSImporter(res, threads, threads)
-	datastore := parser.NewMongoDatastore(res.DB.Session,
+	datastore := parser.NewMongoDatastore(res.DB.Session, res.MetaDB,
```

Comment:
The mongo datastore needs a handle to the meta db to add new databases when they are needed.